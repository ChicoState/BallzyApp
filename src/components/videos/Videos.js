import React, { Component } from 'react';
import NavigationBar from '../videos/navBar';

import {
  StyleSheet,
  Button,
  View,
  Text,
  Alert,
  Navigator,
  Image,
  TouchableHighlight
} from 'react-native';

import Camera from 'react-native-camera';

class Videos extends React.Component {
  static navigationOptions = {
    // Note: By default the icon is only shown on iOS. Search the showIcon option below.
    title: 'Videos',
    tabBarIcon: () => (
      <Image
        source={require('../navigationbar/video48x48.png')}
        style = {{width: 32, height: 32}}
      />
    ),
  };
  render() {
   return(
     <View style={{flex: 1}}>
     <View style={{flex: 2, alignItems: 'center'}}>
     <Text>Hello from videos</Text>
    </View>
    <NavigationBar/>
  </View>
   );
 }
}

const onPress2 = () => {
 Alert.alert('We are working on the upload of video files')
};

const styles = StyleSheet.create({
container: {
 flex: 1,
 flexDirection: 'row',
},
button: {
 justifyContent: 'center',
 alignItems: 'flex-start',
 backgroundColor: 'white',
 borderRadius: 10,
 padding: 10,
},
preview: {
 flex: 1,
 justifyContent: 'flex-end',
   alignItems: 'center'
},
capture: {
 flex: 0,
 backgroundColor: '#fff',
 borderRadius: 5,
 color: '#000',
 padding: 10,
 margin: 40
}
});
export default Videos;
