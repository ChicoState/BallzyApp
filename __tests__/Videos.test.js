'use strict';

import 'react-native';
import React from 'react';
//import React from 'react';
import Videos from '../src/components/videos/Videos';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <Videos />
  ).toJSON();
  expect(tree).toMatchSnapshot();

});
