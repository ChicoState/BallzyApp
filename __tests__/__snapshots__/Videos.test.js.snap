exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 2,
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail">
      Hello from videos
    </Text>
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}>
      <View
        style={
          Array [
            Object {},
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#0C42FD",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
            ]
          }>
          Camera Roll
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 0.2,
        "flexDirection": "row",
      }
    }>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "alignItems": "center",
            "backgroundColor": "#000000",
            "flex": 1,
            "justifyContent": "center",
          },
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}>
      <Image
        source={1} />
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "alignItems": "center",
            "backgroundColor": "#000000",
            "flex": 1,
            "justifyContent": "center",
          },
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}>
      <Image
        source={1} />
    </View>
  </View>
</View>
`;
